platform/vs/docker-sonic-vs/Dockerfile.j2

```diff
@@ -45,7 +45,8 @@ RUN apt-get install -y net-tools \
                        python-scapy \
                        conntrack \
                        iptables \
-                       python3-pip
+                       python3-pip \
+                       libteamdctl0
```

Comment:
we build libteamdctl0 ourselves?

Reply:
You are right! It turns out the package is there, we just need to add it as a runtime dependency so the install respects the ordering. Updated.